src/backy/main.py

```diff
@@ -1,20 +1,18 @@
 # -*- encoding: utf-8 -*-
 
+from prettytable import PrettyTable
 import argparse
 import datetime
 import errno
-import faulthandler
+import humanize
```

Comment:
I'll have to see wether this will collide with isort. This sounds a bit like I should be setting up the pre-commit hooks properly the next time I work on this.